--- CMakeLists.txt
+++ CMakeLists.txt
@@ -60,7 +60,7 @@
     find_library(OGG_LIBRARIES NAMES ogg HINTS ${PC_OGG_LIBRARY_DIRS})
 else()
     find_path(OGG_INCLUDE_DIRS NAMES ogg/ogg.h HINTS ${OGG_ROOT}/include PATH_SUFFIXES ogg)
-    find_library(OGG_LIBRARIES NAMES ogg HINTS ${OGG_ROOT}/lib ${OGG_ROOT}/lib64)
+    find_library(OGG_LIBRARIES NAMES ogg oggd HINTS ${OGG_ROOT}/lib ${OGG_ROOT}/lib64)
 endif()
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(OGG DEFAULT_MSG OGG_INCLUDE_DIRS OGG_LIBRARIES)
